apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -61,15 +60,20 @@ public sealed class PooledSession : IDisposable
         /// Will be <see cref="TransactionOptions.ModeOneofCase.None"/> if <see cref="TransactionId"/>
         /// is null.
         /// </remarks>
-        internal TransactionOptions TransactionOptions { get; }
+        private TransactionOptions TransactionOptions { get; }
+
+        /// <summary>
+        /// The transaction mode for the transaction that is or may be associated with this session.
+        /// </summary>
+        internal TransactionOptions.ModeOneofCase TransactionMode => TransactionOptions.ModeCase;
 
         /// <summary>
         /// The read timestamp of the transaction. May be null.
         /// </summary>
         /// <remarks>
-        /// Will be set iif <see cref="TransactionOptions.ReadOnly"/> have <see cref="Types.ReadOnly.ReturnReadTimestamp"/>
+        /// Will be set iif <see cref="TransactionOptions"/> represents read-only options with <see cref="Types.ReadOnly.ReturnReadTimestamp"/>
```

Comment:
Even if ReturnReadTimestamp is true, this will only return a non-null value if we've already created the transaction, right? Or do we never end up with a pooled session for read-only transaction options but without the transaction already populated?

(I'm more concerned about my own mental model than nit-picking the docs here.)